frontend/public/components/utils/units.js

```diff
@@ -15,7 +15,7 @@ const TYPES = {
     divisor: 1000,
   },
   decimalBytesWithoutB: {
-    units: ['', 'k', 'M', 'G', 'T', 'P', 'E'],
+    units: ['', 'K', 'M', 'G', 'T', 'P', 'E'],
```

Comment:
This change looks incorrect. kB must be specified with a lowercase `k` when setting values like resource limits. `K` causes errors.

Reply:
alright! will change it back with subsequent reviews @spadgett